deploy-agent/setup.py

```diff
@@ -26,10 +26,12 @@
 
 install_requires = [
     "requests==2.20.0",
-    "gevent>=1.0.2,<=1.2.2",
+    "gevent>=1.0.2,<=1.2.2; python_version < '3'",
+    "gevent>=1.0.2,<=1.5.0; python_version >= '3'",
```

Comment:
in the past,  upgrading gevent caused issues. just need to do more testing on the new version of gevent in py3.

Reply:
note the update is only for PY3 and not PY2.  Do you have any links to previous issues?